scripts/code.angularjs.org-firebase/public/robots.txt

```diff
@@ -1,5 +1,6 @@
 User-agent: *
 
-Disallow: /*docs/
+Disallow: /*docs*/
 Disallow: /*i18n/
 Disallow: /*.zip$
+Allow: /snapshot/docs/js/all-versions-data.js
```

Comment:
Apparently wildcards can make the resolution of ordering between rules undefined: https://developers.google.com/search/reference/robots_txt#order-of-precedence-for-group-member-records

Let's keep an eye on this after it goes live...